data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2012](https://gofossilfree.org/about/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Scientists for Future",
+            link: "https://www.scientists4future.org/",
+            keywords: ["science", "grassroots", "education", "ngo"],
+            label: "scientists-for-future",
+        },
+        description: {
+            content: "An interdisciplinary group of scientists working for a sustainable future. They support Fridays for Future and established [Lectures for Future](https://www.scientists4future.org/infomaterial/vortragsreihen/) to push the climate crisis onto the agenda in universities.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "Probably [2019](https://www.scientists4future.org/about/faq/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Europe Beyond Coal",
+            link: "https://www.beyond-coal.eu",
+            keywords: ["coal", "grassroots", "education", "network", "ngo"],
+            label: "europe-beyond-coal",
+        },
+        description: {
+            content: "They pressure governments, investors and insurers to stop supporting extraction and burning of coal in Europe. They track all planned and running [coal plants in the EU](https://beyond-coal.eu/data/) with data on health impact and emissions"
+            +" Europe Beyond Coal is supported by HEAL, WWF, Foundation “Development YES – Open-Pit Mines NO”, Greenpeace, CEE Bankwatch, ClientEarth and CAN Europe",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Bankwatch",
+            link: "https://bankwatch.org/",
+            keywords: ["finance", "divestment", "watchdog", "ngo"],
+            label: "bank-watch",
+        },
+        description: {
+            content: "They investigate the impacts of public finance on communities and help protect their rights and livelihoods."
+            + " Bankwatch monitors the European Bank for Reconstruction and Development, the European Investment Bank, the Asian Infrastructure Investment Bank and the EU in creating future budgets."
+            + " They meet their representatives to make local voices heard to fight for more sustainable and fair investments."
+            + " In addition Bankwatch works on the People's Budget, a proposed budget for the EU that (among others) aims at ending funding for fossil fuels.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Development YES - Open Pit Mines NO",
```

Comment:
I am not sure if we should keep this NGO. It would be hard to classify them. We do not know whether they are still somehow active. We can discus on that.

Reply:
Let's discuss, would keep them in for now until somebody does further research

Reply:
Ok, let's keep it. It is also great to have local organziations from different countries. I would change the wording for the description into something more postive. Take a look.